doc/sphinx/addendum/sprop.rst

```diff
@@ -13,21 +13,18 @@ known as strict propositions) as described in
 :cite:`Gilbert:POPL2019`.
 
 Using :math:`\SProp` may be prevented by passing ``-disallow-sprop``
```

Comment:
```suggestion
Use of :math:`\SProp` may be disabled by passing ``-disallow-sprop``
```

Not incorrect, but IMHO it reads better with "Use of" though I can't cite any grammatical rule for that.